.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.6.3
```

Comment:
Do we need this here?

Reply:
No, I can take that out. I'll add it to .gitignore.